mpi-proxy-split/test/run_all.py

```diff
@@ -0,0 +1,111 @@
+import subprocess
```

Comment:
let's add `#!/usr/bin/env python3` here and add execute permission for the script.